tests/test_partial.py

```diff
@@ -21,12 +21,13 @@
 from inmanta import env
 
 
-@pytest.fixture(scope="session")
-def module_venv(pytestconfig) -> abc.Iterator[env.VirtualEnv]:
+@pytest.fixture(scope="function")
```

Comment:
Doesn't this greatly increase test runtime?

Reply:
I do see why you made the change, I'm just wondering if we should think about another workaround. I've got a few in mind, all of them hacks though.

Reply:
It does make things slower indeed.  This whole PR adds 20 seconds to the test suite's time, I would say it is acceptable?

Reply:
I'm not sure. 20 seconds now is ok, but we haven't got that many v2 module tests yet. Any new tests we add will probably be v2, and at some point we'll have to migrate to v2 when we drop support for v1. So if we can avoid it I think we should. I'll see if I can think of a way to get the best of both worlds.